.eslintrc.json

```diff
@@ -0,0 +1,28 @@
+{
+  "env": {
+    "browser": true,
+    "es6": true,
+    "jest": true
+  },
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "ecmaVersion": 2018,
+    "sourceType": "module"
+  },
+  //"extends": ["airbnb", "plugin:react/recommended"],
```

Comment:
- [ ] Kindly uncomment this. You should use config rules shared by other students. And that line is one of the required rules.